velox/vector/fuzzer/tests/VectorFuzzerTest.cpp

```diff
@@ -96,6 +96,8 @@ TEST_F(VectorFuzzerTest, flatPrimitive) {
       DATE(),
       TIMESTAMP(),
       INTERVAL_DAY_TIME(),
+      LONG_DECIMAL(25, 3),
```

Comment:
Do the precision and scale matter here?
Or will the above implementation randomize the precision and scale?